owncloudApp/src/main/java/com/owncloud/android/presentation/previews/PreviewAudioViewModel.kt

```diff
@@ -37,11 +39,17 @@ class PreviewAudioViewModel(
     private val filterFileMenuOptionsUseCase: FilterFileMenuOptionsUseCase,
     private val contextProvider: ContextProvider,
     private val coroutinesDispatcherProvider: CoroutinesDispatcherProvider,
+    getFileByIdAsStreamUseCase: GetFileByIdAsStreamUseCase,
```

Comment:
You didn't group use cases in parameters here

Reply:
Done 👍